### PR TITLE
The "property" option of DoctrineType was deprecated.

### PR DESCRIPTION
**Pull request by XWB**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.7 |
| Fixed tickets | - |

### CODE REVIEW

File: reference/forms/types/entity.rst [hunk 1]
```diff
@@ -103,6 +103,29 @@ then you can supply the ``choices`` option directly::
 Field Options
 -------------
 
+choice_label
+~~~~~~~~
```
Comment (xabbuh) [inline reference/forms/types/entity.rst]: The underline must match the length of the headline (so you need to add four `~` characters).

File: reference/forms/types/entity.rst [hunk 2]
```diff
@@ -16,7 +16,7 @@ objects from the database.
 |             | - `data_class`_                                                  |
 |             | - `em`_                                                          |
 |             | - `group_by`_                                                    |
-|             | - `property`_                                                    |
+|             | - `choice_label`_                                                |
```
Comment (xabbuh) [inline reference/forms/types/entity.rst]: this should be moved before the `class` option

**Comment by XWB** [comment]
@xabbuh Done!

File: reference/forms/types/entity.rst [hunk 3]
```diff
@@ -12,11 +12,11 @@ objects from the database.
 +-------------+------------------------------------------------------------------+
 | Rendered as | can be various tags (see :ref:`forms-reference-choice-tags`)     |
 +-------------+------------------------------------------------------------------+
+|             | - `choice_label`_                                                |
 | Options     | - `class`_                                                       |
```
Comment (xabbuh) [inline reference/forms/types/entity.rst]: Can you please also move the "Options" to the line above? :)

**Comment by XWB** [comment]
@xabbuh Done!

**Comment by xabbuh** [comment]
:+1: looks great

**Comment by WouterJ** [comment]
Thank you @XWB for fixing this missing bit in the docs!

I've merged your PR and added a versionadded box, indicating that this option is new in Symfony 2.7 and it was previously called `property`, in 2437bd9.

**Comment by XWB** [comment]
:+1:

**Pull request merged**